rhsm-client/src/main/java/org/candlepin/insights/rhsm/client/X509ApiClientFactoryConfiguration.java

```diff
@@ -110,7 +110,8 @@ public boolean usesClientAuth() {
     }
 
     public boolean usesDefaultTruststore() {
-        return getTruststoreFile() == null;
+        return (getTruststoreFile() == null || getTruststoreFile().isEmpty() ||
```

Comment:
I suggest using **StringUtils.isEmpty(getTruststoreFile())** to make this less wordy.

Reply:
DONE and force pushed (9821fd5 to 457cda9)